runtime/server/downloads.go

```diff
@@ -88,6 +93,50 @@ func (s *Server) Export(ctx context.Context, req *runtimev1.ExportRequest) (*run
 	}, nil
 }
 
+func (s *Server) ExportReport(ctx context.Context, req *runtimev1.ExportReportRequest) (*runtimev1.ExportReportResponse, error) {
+	if !auth.GetClaims(ctx).CanInstance(req.InstanceId, auth.ReadMetrics) { // currently all reports are served by mv
+		return nil, ErrForbidden
+	}
```

Comment:
So the idea of the report export API is that it only requires access to the report itself. So instead, I think we can use logic like this on the report resource: https://github.com/rilldata/rill/blob/4d1f20c6a9a048d3bf16292bfce31f732f176b3f/runtime/server/controller.go#L177-L183.

That will trigger the report access logic to be evaluated (which may need some changing for this PR, not sure): https://github.com/rilldata/rill/blob/4d1f20c6a9a048d3bf16292bfce31f732f176b3f/runtime/security.go#L415